.github/workflows/ci.yml

```diff
@@ -65,6 +67,8 @@ jobs:
           cscript configure.js compiler=msvc prefix=C:\thirdparties\libxml2-2.9.10 iconv=no
           nmake -f Makefile.msvc
           nmake -f Makefile.msvc install
+          wget -nv -O boost_1_72_0-msvc-14.2-64.exe https://sourceforge.net/projects/boost/files/boost-binaries/1.72.0/boost_1_72_0-msvc-14.2-64.exe/download
+          boost_1_72_0-msvc-14.2-64.exe /dir="C:\thirdparties\boost-1.72.0" /sp- /verysilent /suppressmsgboxes /norestart
```

Comment:
You might consider having this stage write the boost path to a variable in `$GITHUB_ENV` and then just using that variable in later steps:
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#environment-files

This way future upgrades will not have to change the lines below:

Reply:
Done. Thanks for the suggestion.